pkg/dogstatsd/packets/telemetry.go

```diff
@@ -11,21 +11,9 @@ import (
 
 var (
 	// packet buffer
-	tlmBufferFlushedTimer = telemetry.NewCounter("dogstatsd", "packets_buffer_flush_timer",
-		nil, "Count of packets buffer flush triggered by the timer")
-	tlmBufferFlushedFull = telemetry.NewCounter("dogstatsd", "packets_buffer_flush_full",
-		nil, "Count of packets buffer flush triggered because the buffer is full")
 	tlmChannelSize = telemetry.NewGauge("dogstatsd", "packets_channel_size",
 		nil, "Number of packets in the packets channel")
 
-	// packet pool
-	tlmPoolGet = telemetry.NewCounter("dogstatsd", "packet_pool_get",
-		nil, "Count of get done in the packet pool")
-	tlmPoolPut = telemetry.NewCounter("dogstatsd", "packet_pool_put",
-		nil, "Count of put done in the packet pool")
-	tlmPool = telemetry.NewGauge("dogstatsd", "packet_pool",
-		nil, "Usage of the packet pool in dogstatsd")
-
```

Comment:
Why ?

Reply:
I wanted to talk to you about this. I thought it made sense to have the telemetry counters next to the code that uses them, as opposed to have it in a separate file. If you disagree that's OK, I can move them back. This came up when I was rebasing/transplanting commits, and I decided to make that change. As said, if you prefer to have it all in one place, I'm fine with that.

Reply:
Some of them are used in multiple files. I felt that having everything in one place makes it easier to maintain.

Reply:
I moved it all to telemetry file.